scripts/zones/Jugner_Forest/Zone.lua

```diff
@@ -24,6 +24,11 @@ function onInitialize(zone)
     tpz.conq.setRegionalConquestOverseers(zone:getRegionID())
 
     tpz.helm.initZone(zone, tpz.helm.type.LOGGING)
+
+    local respawnTime = math.random(900, 10800) -- 15 minutes to 3 hours
```

Comment:
If you want to do the spawn windows, they can be done with 
```
local respawnTime(900 * math.random(1, 12)
```
for 12 windows of 15 min each, though that coukd possibly be written better than how I just did to make it obvious whats going on.

Someday maybe we’ll just fee in window size and total tike to a helper function that spit out the right answer.

Reply:
i just followed Roc and Simurgh's lead... are they supposed to have precise windows too?

Reply:
fixed!  thanks teo

Reply:
Wiki says 30 min windows `If Time of death is 6:00pm, the window will begin at 3:05pm the next day and windows will be at the 05 and 35 minute mark.` King Arthro himself should probably also be changed to reflect this. [See Fafnir.](https://github.com/project-topaz/topaz/blob/b72f5d756047375e9123a7cee0a2cb1783c07180/scripts/zones/Dragons_Aery/mobs/Fafnir.lua#L40)